install/grist-install.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://github.com/gristlabs/grist-core
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  curl \
+  sudo \
+  make \
+  gnupg \
+  ca-certificates \
+  mc \
+  python3.11-venv
+msg_ok "Installed Dependencies"
+
+msg_info "Setting up Node.js Repository"
+mkdir -p /etc/apt/keyrings
+curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_20.x nodistro main" >/etc/apt/sources.list.d/nodesource.list
+msg_ok "Set up Node.js Repository"
+
+msg_info "Installing Node.js"
+$STD apt-get update
+$STD apt-get install -y nodejs
+msg_ok "Installed Node.js"
+
+msg_info "Installing yarn"
+$STD npm install -g yarn
+msg_ok "Installed yarn $(yarn --version)"
+
+msg_info "Installing Grist"
+RELEASE=$(curl -s https://api.github.com/repos/gristlabs/grist-core/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+cd /opt
+wget -q https://github.com/gristlabs/grist-core/archive/refs/tags/v${RELEASE}.zip
+unzip -q v$RELEASE.zip
+rm -rf v${RELEASE}.zip
+mv grist-core-${RELEASE} grist
+cd grist
+$STD yarn install
+$STD yarn run build:prod
```

Comment:
The install instructions from https://github.com/gristlabs/grist-core include an additional command, `yarn run install:python` but it is not necessary given I install `python3.11-venv`, which install python 3.11